pandas/tests/indexes/test_common.py

```diff
@@ -388,7 +388,7 @@ def test_asi8_deprecation(self, index):
 @pytest.mark.parametrize("na_position", [None, "middle"])
 def test_sort_values_invalid_na_position(index_with_missing, na_position):
     if isinstance(index_with_missing, (CategoricalIndex, MultiIndex)):
-        pytest.xfail("missing value sorting order not defined for index type")
+        pytest.skip("missing value sorting order not defined for index type")
```

Comment:
hmm i would leave these as xfail and add_marker here. e.g. these are a missing api that we should eventually fix.

cc @jbrockmendel

Reply:
For MultiIndex we get NotImplementedError; this should be certainly be left as xfail. However for CategoricalIndex the computation is successful and sometimes the order is "correct" and sometimes not. I think this should skip instead.

Reply:
Sorry, my remark is about `test_sort_values_with_missing`, the test immediately below this one.